software/cloud-dashboard/react-client/src/components/temp-view/temp-plot.js

```diff
@@ -9,6 +9,8 @@ import Input from 'react-toolbox/lib/input';
 const LINE_COLORS = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00', '#ffff33']
```

Comment:
also, as seen [here](https://github.com/mlp6/ADPL/blob/master/software/cloud-dashboard/react-client/src/components/temp-view.js#L48) the `temp-plot` component has actually been replaced by a more generalized `plot` component which is used to render both the temperature plot and the bucket view plot so these changes should likely go into the `plot` component! My apologies for not promptly removing this legacy component right away. Since it seems like the `plot` component is working fine I can go ahead and remove this component once I ensure there are no lingering dependencies.